en/nodes-hw-testing.md

```diff
@@ -22,9 +22,9 @@ Deployment.
 
 ## Apply a hardware test
 
-To launch a test, on the 'Nodes' page, or within a node's actual view, choose
-action 'Test hardware'. When ready, hit button 'Test machine'. Here, a test is
-being applied to a deployed node:
+To launch a test, select the target machine from the 'Nodes' page and use the
+'Take action' drop-down menu to select 'Test hardware'. When ready, hit button
+'Test machine'. Here, a test is being applied to a deployed node:
```

Comment:
hit **the 'Test Machine'** button?

Reply:
I was attempting to tread lightly on the original text - but you're right, that bugged me too. I'll change it - thanks!